packages/auth/package.json

```diff
@@ -30,7 +30,7 @@
         "react": ">=18.0",
         "react-dom": ">=18.0",
         "react-native": ">=0.70",
-        "react-native-msal": "git+https://github.com/equinor/react-native-msal.git#d6b44b19454226d1bdc30a5d02d424d3e64a5163"
+        "react-native-msal": "*"
```

Comment:
This will point to the react-native-msal package that is in npmjs i think (?).
Is this intended? I was under the impression that we need to use our own fork

Reply:
this is an unchanged line that was temporarily changed to point to a specific commit to test for a fix that was needed for the react-native-msal package that was causing issues with the Code Inspection Github action. After the fix was tested and merged to main this dependency change was reverted.

If there are any changes that are required regarding this dependency I request @tormodAase to give input on this matter

Reply:
Oh, that explains why I saw the diff only for a certain commit.
As it stands right now (also before your PR then), it seems we are indeed pointing to the wrong instance of msal then, correct @tormodAase ?

Reply:
It seems so, as the github action is failing once again due to the same error as before